aspy/refactor_imports/classify.py

```diff
@@ -105,9 +107,20 @@ def classify_import(
     :param text module_name: The dotted notation of a module
     :param tuple application_directories: tuple of paths which are considered
         application roots.
+    :param tuple override_absent_application_modules: tuple of module names
+        that are considered application modules.
     """
+    # Moving future to APPLICATION would cause a SyntaxError.
+    if '__future__' in override_absent_application_modules:
+        raise ValueError(
+            '__future__ cannot be specified in '
+            'override_absent_application_modules as it must occur at the '
+            'beginning of a file',
+        )
```

Comment:
hmm so I was thinking instead to move the if statement you added into the chain of ifs below the `__future__` check

Reply:
Of course, much simpler. Done